### PR TITLE
portfolio accessibility

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
Updated Links, Header Tag level, and Contrast of Colours for more Visibility
The report for this pull request contains the following changes:

Pages title:  
1. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1

Image text alternative: no issue found

Text headings:
1. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L47
2. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L112
3. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L215
4. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L215
5. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L258
6. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L258
7. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L47

Updated the Links :
1. [twitter](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L341) 
2. [linkedin](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L331)
3. [GitHub](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L326)
4. [media](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L347)
5. [angel](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L336)

Added Labels to form :
1.  https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R371
2. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L405
3. https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R390

Color Contrast :
1. [contrast changed](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4L30)
2. [contrast changed](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4L47)
3. [contrast changed](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4L115)
4. [contrast changed](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4L317)
5. [contrast changed](https://github.com/ismailakinkunmi/portfolio-site/pull/8/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4L608)

Resize
Interaction
Moving content
Multimedia
The basic structure of the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,14 +95,16 @@ <h5 class="connect">LET’S CONNECT</h5>
           <li>
             <img
               src="img/card Portfolio.png"
-              alt="card"
+              alt
+              text="nature card"
```
Comment (VanessaAoki) [inline index.html:99]: - [ ] Good job adding the `alt` attribute to your images 👏  unfortunately, this is not the correct syntax to implement an alt text to an image, it should be:
```
 <img src="img/card Portfolio.png" class="desktop-images" alt="Tonic website screenshot">
```
- [ ] You should consider double-checking all the images and their `alt` attribute in the project, to guarantee they all have the correct syntax and meaningful text.
> For more info on alt text, you read [this article](https://axesslab.com/alt-texts/).

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# portfolio-setup
+# Responsive Portfolio
```
Comment (VanessaAoki) [inline README.md:1]: - [ ] [OPTIONAL]: You should consider editing this heading to something more personal. For instance "_Your name's_ portfolio"

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @ismailakinkunmi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Keyboard interaction works perfectly, good project structure with proper headings, contrast ratio, and resizable text 🔝 

## Required Changes ♻️
_Check the comments under the review._
- [ ] According to the requirements, you should have a report in your PR that must include:

image
- [ ] According to the form accessibility guide, all form controls have a label associated with them, using `label`, `for`, and `id`. You can read more about it [here](https://www.w3.org/WAI/test-evaluate/preliminary/#labelschecks).


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (VanessaAoki) [reply README.md]: According to the requirement, your report should have a list of everything that was checked, as if it was a checklist, having it "no changes required" for everything is not enough.

Comment (ismailakinkunmi) [reply README.md]: ok,

**Review by Cathella — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑

### Hi @ismailakinkunmi 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [x] [REQUIRED] As mentioned by the previous reviewer please consider adding _***(e.g. "Page titles - no issues found")***_ if an accessibility check was found okay in the summary of your pull request  😄.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @ismailakinkunmi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**